dotnet/src/Connectors/Connectors.Amazon/Bedrock/Core/Models/AI21Labs/AI21JambaResponse.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.Json.Serialization;
+
+namespace Microsoft.SemanticKernel.Connectors.Amazon.Core;
+
+/// <summary>
+/// AI21JambaResponse objects for Bedrock Runtime actions.
+/// </summary>
+internal static class AI21JambaResponse
+{
+    /// <summary>
+    /// AI21 Text Generation Response object (from Invoke).
+    /// </summary>
+    [Serializable]
+    internal class AI21TextResponse
+    {
+        /// <summary>
+        /// A unique ID for the request (not the message). Repeated identical requests get different IDs. However, for a streaming response, the ID will be the same for all responses in the stream.
+        /// </summary>
+        [JsonPropertyName("id")]
+        public string? Id { get; set; }
+        /// <summary>
+        /// One or more responses, depending on the n parameter from the request.
+        /// </summary>
+        [JsonPropertyName("choices")]
+        public List<Choice>? Choices { get; set; }
+        /// <summary>
+        /// The token counts for this request. Per-token billing is based on the prompt token and completion token counts and rates.
+        /// </summary>
+        [JsonPropertyName("usage")]
+        public Usage? Use { get; set; }
+        /// <summary>
+        /// The members for the Choice class as required by AI21 Labs Jamba.
+        /// </summary>
+        [Serializable]
+        internal class Choice
```

Comment:
Avoid having 2 levels of `inner classes`, keep all within `AI21JambaResponse` as 1 level deep at most for simplicity.

```suggestion
        internal class Choice
```